examples/test_vpc_msk_setup/main.tf

```diff
@@ -0,0 +1,74 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+# Create a VPC
+resource "aws_vpc" "test_vpc" {
+  cidr_block           = var.vpc_cidr
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+  tags = {
+    Name = "msk_pl_test_vpc"
+  }
+}
+
+# Create subnets
+resource "aws_subnet" "test_subnet" {
+  count                   = length(var.subnet_cidrs)
+  vpc_id                  = aws_vpc.test_vpc.id
+  cidr_block              = var.subnet_cidrs[count.index]
+  availability_zone       = element(var.availability_zones, count.index)
+  map_public_ip_on_launch = true
```

Comment:
I don't think you want this to be true if the only access is going to be from privatelink.